src/libs/actions/PaymentMethods.js

```diff
@@ -1,14 +1,33 @@
+import {createRef} from 'react';
 import lodashGet from 'lodash/get';
 import Onyx from 'react-native-onyx';
 import ONYXKEYS from '../../ONYXKEYS';
 import * as API from '../API';
 import CONST from '../../CONST';
-import ROUTES from '../../ROUTES';
 import Growl from '../Growl';
 import * as Localize from '../Localize';
 import Navigation from '../Navigation/Navigation';
 import * as CardUtils from '../CardUtils';
 
+/**
+ * Sets up a ref to an instance of the KYC Wall component.
+ */
+const kycWallRef = createRef();
+
+/**
+ * When we successfully add a payment method or pass the KYC checks we will continue with our setup action if we have one set.
+ */
+function continueSetup() {
+    if (!kycWallRef.current || !kycWallRef.current.continue) {
+        Navigation.goBack();
+        return;
+    }
+
+    // Close the view on success and continue with setup
+    Navigation.goBack();
```

Comment:
Which view are we closing here?

Reply:
I can clarify this in the comment. But in this case it will either be the add debit card, add bank account, or enable payments screens.